src/components/entrybutton/EntryButton.styled.ts

```diff
@@ -15,7 +15,7 @@ export const Primary = styled.button`
   text-align: center;
   text-align: center;
   margin: 10px auto;
-  width: 90%;
+  width: ${(props) => props.width}px;
```

Comment:
A button should have a dynamic CSS size most of the time to facilitate the text and icons inside

Reply:
This was just for a test to see if you'd be able to use a  slider in StoryBook